jfr-streaming/src/parallelGcTest/java/io/opentelemetry/contrib/jfr/metrics/PsGcDurationMetricTest.java

```diff
@@ -18,35 +18,16 @@
 import org.assertj.core.api.ThrowingConsumer;
 import org.junit.jupiter.api.Test;
 
-class GCDurationMetricTest extends AbstractMetricsTest {
+class PsGcDurationMetricTest extends AbstractMetricsTest {
   static class AttributeCheck implements ThrowingConsumer<PointData> {
     @Override
     public void acceptThrows(PointData pointData) throws Throwable {
-      // Each point must have attributes of one of the following variation:
-      // First sort by Major/Minor, then by GC.
       if (pointData.getAttributes().get(ATTR_ACTION).equals(END_OF_MINOR_GC)) {
-        if (garbageCollectors.contains("G1 Young Generation")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(
-                  Attributes.of(ATTR_GC, "G1 Young Generation", ATTR_ACTION, END_OF_MINOR_GC));
-        } else if (garbageCollectors.contains("PS Scavenge")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(Attributes.of(ATTR_GC, "PS Scavenge", ATTR_ACTION, END_OF_MINOR_GC));
-        } else if (garbageCollectors.contains("Copy")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(Attributes.of(ATTR_GC, "Copy", ATTR_ACTION, END_OF_MINOR_GC));
-        }
+        assertThat(pointData.getAttributes())
+            .isEqualTo(Attributes.of(ATTR_GC, "PS Scavenge", ATTR_ACTION, END_OF_MINOR_GC));
       } else {
-        if (garbageCollectors.contains("G1 Old Generation")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(Attributes.of(ATTR_GC, "G1 Old Generation", ATTR_ACTION, END_OF_MAJOR_GC));
-        } else if (garbageCollectors.contains("PS MarkSweep")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(Attributes.of(ATTR_GC, "PS MarkSweep", ATTR_ACTION, END_OF_MAJOR_GC));
-        } else if (garbageCollectors.contains("MarkSweepCompact")) {
-          assertThat(pointData.getAttributes())
-              .isEqualTo(Attributes.of(ATTR_GC, "MarkSweepCompact", ATTR_ACTION, END_OF_MAJOR_GC));
-        }
+        assertThat(pointData.getAttributes())
+            .isEqualTo(Attributes.of(ATTR_GC, "PS MarkSweep", ATTR_ACTION, END_OF_MAJOR_GC));
       }
     }
   }
```

Comment:
There's a comment below that is no longer true:
```
// TODO: Need a reliable way to test old and young gen GC in isolation.
```

Reply:
I left that comment in because, although we can reliably test different collectors independently, there still isn't any code that enforces young vs old gen collection for GCs that divide up the heap.